### ISSUE TITLE
Syntactic extensions can cause module parse failure

### ISSUE THREAD

**Issue by avieth** [gibiansky/IHaskell#1114]
A module such as

```Haskell
{-# LANGUAGE ScopedTypeVariables #-} 

module Foo where

bar :: forall a . a -> a
bar x = x
```

cannot be loaded in an IHaskell notebook. It shows a parse failure, so apparently the module parser doesn't enable the extension. But if `:set -XScopedTypeVariables` is entered before trying to `:load` the file, it works.

**Comment by vaibhavsagar** [comment]
Thanks for raising this! Fixed in afa0de07b8fcb45374b0f224bcd0ae9092811a02.

**Comment by avieth** [comment]
> Thanks for raising this! Fixed in afa0de0.

Thanks for fixing it so fast!
